content/docs/user-guide/external-dependencies.md

```diff
@@ -149,12 +151,12 @@ $ dvc import-url https://data.dvc.org/get-started/data.xml
 Importing 'https://data.dvc.org/get-started/data.xml' -> 'data.xml'
 ```
 
-The command above creates the <abbr>import stage</abbr> (DVC-file)
-`data.xml.dvc`, that uses an external dependency (in this case an HTTPs URL).
+The command above creates the import `.dvc` file `data.xml.dvc`, that contains
+an external dependency (in this case an HTTPs URL).
 
 <details>
 
-### Expand to see resulting DVC-file
+### Expand to see resulting .dvc file
```

Comment:
we can now use inline `code`

Reply:
Yes, thanks. But actually this may look better? In any case we have a few other similar headers in other docs so I can update them all in a separate small PR if you like how it looks with back quotes more.

## without .dvc file

## with `.dvc` file

Reply:
I don't know, I prefer them highlighted. It feels consistent.

Reply:
That PR changed all the `details` triggers that have "DVC-file" to `.dvc` file.

Here's the `details` that are like this in master right now

**content/docs/user-guide/external-dependencies.md**
157:### Expand to see resulting `.dvc` file
196:### Expand to see resulting `.dvc` file

The remaining instances of "DVC-file" in headings only exist in the Heartbeat blog posts.

If we decide "DVC-file" looks better, we'll have to change them all back in a PR.

Reply:
OK changed for this one file. Will send all the other instances in a separate PR when this one is merged.

Reply:
@rogermparent "DVC-file" is a deprecated term 🙂

Reply:
My mistake, I get what you mean now!